data/yam/autoyast/support_images/sled15sp5_install_all_patterns_x86_64.xml

```diff
@@ -0,0 +1,203 @@
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <bootloader t="map">
+    <global t="map">
+      <timeout t="integer">45</timeout>
+    </global>
+  </bootloader>
+  <firewall t="map">
+    <enable_firewall t="boolean">true</enable_firewall>
+    <start_firewall t="boolean">true</start_firewall>
+    <zones t="list">
+      <zone t="map">
+        <interfaces t="list">
+          <interface>eth0</interface>
+        </interfaces>
+        <name>public</name>
+        <ports t="list">
+          <port>22/tcp</port>
+          <port>30000-50000/tcp</port>
```

Comment:
Is there any reason for setting port `30000-50000`?

Reply:
I copy from the generated profile with general installation job, do you have any suggestions on this? Or remove it from this profile?

Reply:
I copy from the generated profile with general installation job, do you have any suggestions on this? Or remove it from this profile?

Reply:
I removed it and shorted the profile, thank you.